tools/rapids-github-run-id

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# A utility script that examines environment variables provided
+# by GitHub Actions to print out a Github Workflow Run ID where the expected artifact
+# should be.
+set -euo pipefail
+source rapids-constants
+export RAPIDS_SCRIPT_NAME="rapids-github-run-id"
+
+case "${RAPIDS_BUILD_TYPE}" in
+  pull-request)
+    run_id=$(gh run list --repo "${RAPIDS_REPOSITORY}" --branch "${RAPIDS_REF_NAME}" --commit "${RAPIDS_SHA}" --json databaseId --jq '.[0] | .databaseId')
+    ;;
+  branch)
+    run_id=$(gh run list --repo "${RAPIDS_REPOSITORY}" --branch "${RAPIDS_REF_NAME}" --commit "${RAPIDS_SHA}" --json databaseId --jq '.[0] | .databaseId')
+    ;;
+  nightly)
+    run_id=$(gh run list --repo "${RAPIDS_REPOSITORY}" --branch "${RAPIDS_REF_NAME}" --commit "${RAPIDS_SHA}" --json databaseId --jq '.[0] | .databaseId')
+    ;;
+  *)
+    rapids-echo-stderr "please pass a valid RAPIDS_BUILD_TYPE"
+    exit 1
+    ;;
+esac
```

Comment:
```suggestion
case "${RAPIDS_BUILD_TYPE}" in
  pull-request | branch | nightly)
    run_id=$(gh run list --repo "${RAPIDS_REPOSITORY}" --branch "${RAPIDS_REF_NAME}" --commit "${RAPIDS_SHA}" --json databaseId --jq '.[0] | .databaseId')
    ;;
  *)
    rapids-echo-stderr "please pass a valid RAPIDS_BUILD_TYPE"
    exit 1
    ;;
esac
```

I think we can simplify this case statement to do the one command for any of the valid build types